edb/pgsql/deltadbops.py

```diff
@@ -211,6 +226,15 @@ def is_natively_inherited(self):
         """Determine if this constraint can be inherited natively."""
         return self._type == 'check'
 
+    def requires_triggers(self):
+        return (
+            not self.is_natively_inherited()
```

Comment:
I assume `is_natively_inherited` isn't needed anymore?

Reply:
It's a bit misleadingly named now, but the check is still valid.  I'll adjust.